index.js

```diff
@@ -1,31 +1,31 @@
-module.exports = function tabto(input) {
-    if (!input instanceof HTMLInputElement) {
-        console.error('tabto() expect a DOM Input element.');
-        return;
-    }
+/* global HTMLInputElement */
 
-    if (!input.dataset.tabTarget) {
-        console.error('tabto() expect the DOM Input element to have a "data-tab-target" attribute.');
-        return;
-    }
+module.exports = function tabto (input) {
+  if (!(input instanceof HTMLInputElement)) {
+    console.error('tabto() expect a DOM Input element.')
+    return
+  }
 
-    var maxLength = input.maxLength;
-    var next = document.querySelector(input.dataset.tabTarget);
-    var currentValue = input.value;
+  if (!input.dataset.tabTarget) {
+    console.error('tabto() expect the DOM Input element to have a "data-tab-target" attribute.')
+    return
+  }
 
-    if (!next || !maxLength) {
-        console.error('tabto() a valid target and a maxLength on the provided input.');
-        return;
-    }
+  const maxLength = input.maxLength
+  const next = document.querySelector(input.dataset.tabTarget)
+  if (!next || !maxLength) {
+    console.error('tabto() a valid target and a maxLength on the provided input.');
```

Comment:
Looks like the semicolons have been removed everywhere but not here?

Reply:
Standard not uses `;`, Js will do it for you with no performance sacrifice. I installed Standard and linted `index.js` file